README.md

```diff
@@ -1,48 +1,46 @@
 
-## Supported devices (running firmware 1.8.0 or later) *:
-- Sonoff S20 **
-- Sonoff S26
+## Tested Devices
+ - Sonoff Basic
+ - Sonoff RF
+ - Sonoff S26
+ - Sonoff T1 UK
+
+## Expected Supported devices:
+- Sonoff S20 
 - Sonoff S85
 - Sonoff T1 UK/US/EU 1 Gang
 - Sonoff S30
 - Sonoff S31 Lite
-- Sonoff Basic
-- Sonoff RF
 - Sonoff Slampher
 - Sonoff SV
 - Sonoff Touch EU/US
-- Any other Sonoff / eWeLink device running firmware version 1.8.0 or later
+- Any other Sonoff / eWeLink single channel device
 
-\* I currently only own a Sonoff Basic so haven't actually tested with these devices. However, I am very confident that if your device works with the eWeLink app in LAN Mode, we can get it working with this component - we might need a bit of joint investigation (e.g. `tcpdump` of communication from app) first to get it working!
+## Unsupported Devices*
+ - Multi outlets devices
 
-\*\* Some Sonoff devices seem to not boot directly into LAN mode despite internet access being blocked; there is an ongoing issue open to investigate this and attempt to automate the process of forcing a Sonoff into LAN mode.
-Until this is implemented / resolved, for some devices you may need to let it first boot with internet access, then block it's connectivity to get it into LAN mode (which is quite inconvenient if your device cannot be permanently powered).
+However, I am very confident that if your device works with the eWeLink app in LAN Mode, we can get it working with this component - we might need a bit of joint investigation (e.g. `tcpdump` of communication from app) first to get it working!
 
 ## What is LAN Mode?
-Since mid 2018, the firmware Itead have shipped with Sonoff devices has provided a feature called "LAN Mode" which allows the device to be controlled directly on the local network using a WebSocket connection on port 8081.
+Since mid 2018, the firmware Itead have shipped with Sonoff devices has provided a feature called "LAN Mode" which allows the device to be controlled directly on the local network using a WebSocket or HTTP REST connection on port 8081.
 
-The feature is designed to only be used when there is no connection to the Itead cloud servers (e.g. if your internet connection is down, or their servers are down). As such, it is only enabled when the device is connected to your WiFi network, but unable to reach the Itead servers. You can find out more about the feature in the [Itead FAQ page](https://help.ewelink.cc/hc/en-us/articles/360007134171-LAN-Mode-Tutorial).
+Whilst older devices only dropped into LAN moe when the internet was unavailabel, the latest V3 firmware are designed to be used in LAN mode predominantly. Note: the eWeLink app doesn't show this very well and when you turn it to LAN mode, it doesn't report that it connect to V3 devices in this mode, so many users are unaware of this and believe the device to be connected via internet, however there is an icon against the device showing a LAN icon when using LAN mode.
```

Comment:
A few minor typos / sentence structure tweaks here - happy to tweak myself if merged though.

Reply:
I've made some improvements, but please feel free to refine